### PR TITLE
Issue 10 indexmap

### LINKED ISSUE #10: Keep imports in order
When outputting the exports, we take care that they get exported in the same order that the tool receives them. This makes sure that when generating the TS bindings for the same Rust source module two times in a row, you get the exact same output.

The imports, on the other hand, don't follow a specific order, and may vary depending on the mood of the HashMap. We need to sort the imports before outputting them to the file.

### PR DESCRIPTION
**Pull request by greyblake**
Fixes #10 .
Second try for #10.
Tested against the main code base.

### CODE REVIEW

File: ts_export/src/process.rs [hunk 1]
```diff
@@ -167,6 +167,8 @@ impl ProcessModule {
             })
             .collect();
 
+        println!("{:?}", imports);
```
Comment (AlisCode) [inline ts_export/src/process.rs:170]: Can you remove that please ?

**Review by AlisCode — CHANGES_REQUESTED** [review]
Looks good, just need to remove the println (I'd typically replace that with `log::info!` or `log::debug!` but I dont see the added value. 

I don't think that there's any more tests to be added, indexmap is statically solving the ordering problem, and I'm unclear on how to test that sort of behaviour. 

Cheers

File: ts_export/src/process.rs (hunk 1 above)
Comment (greyblake) [reply ts_export/src/process.rs]: Shaaaame on me.
Addressed in 36aa6c1f48965414485fc0435622dbb8df737ad6

**Comment by AlisCode** [comment]
Thanks! Looks good now :)

**Pull request merged**